oak/proto/storage.proto

```diff
@@ -50,29 +50,36 @@ message DeleteResponse {
 }
 
 message BeginRequest {
-  bytes data_store_id = 1;
+  bytes storage_id = 1;
 }
 
 message BeginResponse {
   bytes transaction_id = 1;
 }
 
 message CommitRequest {
-  bytes transaction_id = 1;
+  bytes storage_id = 1;
+  bytes transaction_id = 2;
 }
 
 message CommitResponse {
 }
 
 message RollbackRequest {
-  bytes transaction_id = 1;
+  bytes storage_id = 1;
+  bytes transaction_id = 2;
 }
 
 message RollbackResponse {
 }
 
 // Wrapper message used to send requests via the StorageChannel interface.
 message StorageOperationRequest {
+  // The name of the storage on which to perform the operation.
+  // The StorageChannel generates a deterministic name-based (version 5) UUID
+  // from this value and the Oak Module to use as the storage identifier.
```

Comment:
(I didn't really follow this, but that might be lack of background)

Reply:
I'm also still not entirely sure what this is used for or whether it's necessary / nice to have (and why).

Reply:
In what sense does the StorageChannel generate something from this value __and the Oak Module__? Do you mean the name of the module? or the hash of the code? or something else?

Reply:
Clarified.